src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,45 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ELEMENTS = 10;
+    private final K[] keys;
+    private final V[] values;
+    private int now;
```

Comment:
not informative naming
```suggestion
    private int size;
```